.github/workflows/labeler-community.yml

```diff
@@ -0,0 +1,12 @@
+# https://github.com/actions/labeler
+
+name: Community PR Labels
+on:
+- pull_request_target
+
+jobs:
+  labeler:
+    if: ${{ github.event.pull_request.head.repo.full_name != github.repository }}
+    uses: ./.github/workflows/labeler-reusable.yml
+    secrets:
+      repo-token: "${{ secrets.DD_CHANGELOG_CHECK_TOKEN }}"
```

Comment:
Can you confirm we need this secret in the "labeler community" workflow?

Reply:
Yes, we'll need the secret for the `write` permission in order for the workflow to add new labels to the PR